### ISSUE TITLE
[Bug][BanyanDB-UI] Overlay display wrongly

### ISSUE THREAD

**Issue by hanahmily** [apache/skywalking#11978]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

BanyanDB (apache/skywalking-banyandb)

### What happened

When the server returns an error, the overlay still displays to cover all components.

<img width="1350" alt="image" src="https://github.com/apache/skywalking/assets/1960537/c9547b6d-10bf-400f-9e65-180df0ed25dc">

### What you expected to happen

The overlay should disappear.

### How to reproduce

1. Open UI
2. Close the server to stimulate
3. Click any resources in measure or stream.

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Muyu-art** [comment]
导师你好，我是范茂茂，我可以先解决这个issue

**Comment by wu-sheng** [comment]
Please use English on GitHub.

**Comment by Muyu-art** [comment]
Hi, mentor, I`m Fan Maomao, I can solve this issue~ Can assign this to me?

**Comment by Muyu-art** [comment]
https://github.com/apache/skywalking/assets/72800755/593669a4-b665-439a-9f98-ae998c6b1e6e

I have solved this issue, here is a brief video to introduce the outcome.

**Comment by wu-sheng** [comment]
This video seems not to play properly.

**Comment by Muyu-art** [comment]
https://github.com/apache/skywalking/assets/72800755/bfab50e3-ec7b-4aa4-bd7d-114b0eabd532

**Comment by wu-sheng** [comment]
I have tried to download the file, still not working.

**Comment by Muyu-art** [comment]
OK, I will try other video formats

**Comment by Muyu-art** [comment]
[2024-03-06 15-36-47.webm](https://github.com/apache/skywalking/assets/72800755/f8d0164f-80cf-4e23-8407-95edc3bb9883)

I closed the back end and reproduced the above problem. Troubleshooting revealed that in the getGroupList function, the mask could not be closed because the data could not be requested and no subsequent processing was done. I added $loadingClose() at the end of the function, which solved the problem.
